src/__tests__/App.test.js

```diff
@@ -35,8 +35,7 @@ describe("App.js", () => {
   beforeEach(() => {
```

Comment:
We should update this test class to also assert on jails-related calls below.

Reply:
Done